steering-and-technical-committee-charter.md

```diff
@@ -0,0 +1,226 @@
+   violations and maintain confidentiality.
+1. Approval of logo changes, significant website updates and marketing campaigns.
+1. Establish processes regarding project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+1. Review security reports; work with the appropriate technical leads to accept
+   or reject the report; maintain the private nature of such reports until
+   disclosed to the broader community.
+1. Act as the final escalation point and decider for any disputes, issues,
+   clarifications, or escalations within the project scope. This includes delegate appropriate authority to trusted individuals.
+1. Resolve technical issues, technical disagreements and escalations within the project.
+1. Provide guidance to, and be informed by the release team for milestone releases, and maintainers to define milestones. 
+1. Decide on technical issues that affect multiple repositories when working with repository maintainers, to ensure cohesive and consistent decisions are made. 
+1. Approve the creation/deletion of GitHub repositories, along with other high-level administrative issues around GitHub and our other tools.
+1. Establish and maintain the overall technical governance guidelines and processes for the project including a final escalation path.
+1. Foster an environment for a healthy and happy community of developers and contributors.
+
+## Committee meetings
+
+STC meets monthly. Meetings are open to public and held online.
+
+Given the private nature of some of these discussions, for example privacy, private
+emails to the committee, code of conduct violations, escalations, disputes
+between members, security reports, some meetings are held in private. Committee meetings can be made closed by a simple majority vote of the STC members, but closed meetings do not count towards the required meeting frequency. The governance committee will inform the community if a particular meeting is closed, as well as the general reasons for this, prior to the meeting taking place.
+
+Public meetings are recorded and the [recordings](XX) and [meeting notes](XXX) are available publicly. If a member misses more than 50% of all meetings held within a six month period then they will be removed from the committee and a special election will be held. 
+
+Questions and proposals for changes to governance are posted as [issues in the community repository](https://github.com/Dapr/community/issues), and the STC invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+To bootstrap the STC 5 individuals were identified to provide the initial process that can bootstrap the remainder of the process.The bootstrap committee will be replaced by an elected Dapr Committee. A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr repository.
```

Comment:
Change `A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr repository` to `A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr org`.

Reply:
Updated